packages/discovery-react-components/src/components/DocumentPreview/components/SimpleDocument/SimpleDocument.tsx

```diff
@@ -32,7 +34,8 @@ export const SimpleDocument: FC<Props> = ({
   highlight,
   setLoading,
   setHideToolbarControls,
-  cannotPreviewMessage = 'Cannot preview document'
+  cannotPreviewHeader = 'Can\'t preview document',
+  cannotPreviewMessage = 'Try the JSON tab for a different view of this document\'s data.'
```

Comment:
This is part of our API, so what you are proposing is a breaking change.

I suggest keeping the "header" as `cannotPreviewMessage` and name the other prop as something like `cannotPreviewMessage2`